packages/mobile/src/store/oauth/reducer.ts

```diff
@@ -27,6 +31,14 @@ type InstagramInfo = {
   instagramId?: any
 }
 
+type TikTokInfo = {
+  uuid: string
+  profile: TikTokProfile
+  profileImage: any
```

Comment:
Any thoughts here?

Reply:
Yeah I updated it now to a more specific type, was just `any` because that was the case for twitter and ig. I may go through those flows to improve this stuff too after I get the tiktok stuff in a good spot